package.json

```diff
@@ -45,58 +45,56 @@
     "react-dom": "^15.0.0 || ^16.0.0"
   },
   "dependencies": {
-    "lodash": "~4.17.4",
+    "lodash": "^4.17.20",
     "prop-types": "^15.6.0",
     "raf": "^3.4.0",
-    "react-transition-group": "^2.5.0"
+    "react-transition-group": "^4.0.0"
   },
   "devDependencies": {
-    "@babel/cli": "^7.1.0",
-    "@babel/core": "^7.0.0",
-    "@babel/plugin-proposal-class-properties": "^7.0.0",
-    "@babel/plugin-proposal-decorators": "^7.0.0",
-    "@babel/plugin-proposal-export-default-from": "^7.0.0",
-    "@babel/plugin-proposal-export-namespace-from": "^7.0.0",
-    "@babel/plugin-proposal-function-bind": "^7.0.0",
-    "@babel/plugin-proposal-object-rest-spread": "^7.0.0",
-    "@babel/plugin-transform-runtime": "^7.0.0",
+    "@babel/cli": "^7.12.0",
+    "@babel/core": "^7.12.0",
+    "@babel/plugin-proposal-class-properties": "^7.12.0",
+    "@babel/plugin-proposal-decorators": "^7.12.0",
+    "@babel/plugin-proposal-export-default-from": "^7.12.0",
+    "@babel/plugin-proposal-export-namespace-from": "^7.12.0",
+    "@babel/plugin-proposal-function-bind": "^7.12.0",
+    "@babel/plugin-proposal-object-rest-spread": "^7.12.0",
+    "@babel/plugin-transform-runtime": "^7.12.0",
     "@babel/polyfill": "^7.0.0",
-    "@babel/preset-env": "^7.0.0",
-    "@babel/preset-react": "^7.0.0",
-    "@babel/runtime": "^7.0.0",
+    "@babel/preset-env": "^7.12.0",
+    "@babel/preset-react": "^7.12.0",
+    "@babel/runtime": "^7.12.0",
     "babel-eslint": "^10.0.0",
+    "babel-jest": "^26.6.3",
     "babel-loader": "^8.0.0",
     "babel-plugin-lodash": "^3.3.0",
-    "chai": "^4.1.2",
-    "chai-enzyme": "^0.8.0",
-    "chai-spies": "^0.7.1",
     "cross-env": "^5.0.5",
-    "enzyme": "^3.1.0",
-    "enzyme-adapter-react-16": "^1.0.2",
+    "enzyme": "^3.11.0",
+    "enzyme-adapter-react-16": "^1.15.0",
     "eslint": "^4.8.0",
     "eslint-config-airbnb": "^16.0.0",
     "eslint-plugin-import": "^2.7.0",
     "eslint-plugin-jsx-a11y": "^6.0.2",
     "eslint-plugin-react": "^7.4.0",
+    "jest": "^26.6.3",
     "json-loader": "^0.5.7",
-    "karma": "^3.0.0",
-    "karma-chai": "^0.1.0",
-    "karma-chrome-launcher": "^2.2.0",
-    "karma-coverage": "^1.1.0",
-    "karma-coveralls": "^2.1.0",
-    "karma-firefox-launcher": "^1.1.0",
-    "karma-mocha": "^1.3.0",
-    "karma-sourcemap-loader": "^0.3.7",
-    "karma-webpack": "^4.0.0-rc.2",
-    "mocha": "^4.0.1",
     "pre-commit": "^1.1.3",
     "react": "^16.0.0",
     "react-addons-test-utils": "^15.6.2",
     "react-dom": "^16.0.0",
-    "webpack": "^4.20.0",
-    "webpack-bundle-analyzer": "^2.11.0",
-    "webpack-cli": "^3.1.0",
-    "webpack-dev-server": "^3.1.9"
+    "request": "^2.88.2",
+    "webpack": "^4.44.2",
+    "webpack-bundle-analyzer": "^4.3.0",
+    "webpack-cli": "^4.2.0",
+    "webpack-dev-server": "^3.11.0"
+  },
+  "jest": {
+    "transform": {
+      "\\.[jt]sx?$": "babel-jest"
```

Comment:
It's possible to avoid specifying `transform` and also the `babel-jest` dep:

https://jestjs.io/docs/en/getting-started#using-babel

>  Note: babel-jest is automatically installed when installing Jest and will automatically transform files if a babel configuration exists in your project. To avoid this behavior, you can explicitly reset the transform configuration option:

Reply:
Yes I have tried but didn t find why it was not working 🤷